content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -79,72 +72,97 @@ npm install
+3. Select the **Settings** tab in the page of the Heroku Postgres add-on.
+4. Click **View Credentials**.
+5. From the **URI** field, copy the entire connection string.
 
-Set the `DATABASE_URL` environment variable locally so you can create the database schema and Prisma can access the database to introspect:
+**Checkpoint:** The Heroku connection string has the following format:
+
+```
+postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+```
+
+When you next set the `DATABASE_URL` environment variable, you need to change the beginning string from `postgres` to `postgresql`.
+
+## 4. Set the DATABASE_URL environment variable locally
+
+Set the `DATABASE_URL` environment variable locally. The `prisma.schema` file uses this variable so that Prisma can access the database to introspect:
 
 ```terminal
 export DATABASE_URL="postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
 ```
 
-> **Note:** you will need the `DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
+> **Note:** You need the `DATABASE_URL` environment variable for the subsequent steps. Set it in all terminal sessions related to this project.
 
 You need to replace the uppercase placeholders with your database credentials, e.g.:
 
 ```terminal
 postgresql://janedoe:randompassword@yourpostgres.compute-1.amazonaws.com:5432/yourdbname
 ```
 
-## 4. Create the database schema
+Or simply copy and paste the connection string you get from your cloud database provider.
 
-To create your database schema, run the `schema.sql` from the example code as follows:
+## 5. Create a shadow database in your cloud provider and get the connection string
 
-```terminal
-psql $DATABASE_URL -f schema.sql
-```
+In the later stages of this guide, you will run the `prisma migrate dev --name init` command to map the data model (that is pre-defined for this project in the `schema.prisma` file) to the database. As you develop your app, you continuously need to run `prisma migrate dev` whenever you make a change to `schema.prisma` so that the changes are migrated to the database.
 
-**Checkpoint:** `psql $DATABASE_URL -c "\dt"` should show the list of tables:
+This is part of the development cycle of your app and for all such operations Prisma requires the availability of a [shadow database](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database).
 
-```text
-      List of relations
-Schema |  Name   | Type  |     Owner
---------+---------+-------+----------------
-public | Post    | table | janedoe
-public | Profile | table | janedoe
-public | User    | table | janedoe
-```
+Cloud database providers do not allow the automated creation of a shadow database. To enable a shadow database, you need to create a new database manually in your provider.
+
+After you create a new database, get its connection string.
+
+## 4. Set the SHADOW_DATABASE_URL environment variable locally
```

Comment:
Wrong list number (should be 6, after 5)

Reply:
These are fixed in my next commit.

Reply:
I've fixed the numbering of all sections now. Thanks for catching this @janpio